examples/11/example11.go

```diff
@@ -63,8 +63,9 @@ func (r *MyCustomResponseWriter) Typing() {
 }
 
 // Reply send a attachments to the current channel with a message
-func (r *MyCustomResponseWriter) Reply(message string, options ...slacker.ReplyOption) {
+func (r *MyCustomResponseWriter) Reply(message string, options ...slacker.ReplyOption) error {
 	rtm := r.botCtx.RTM()
 	event := r.botCtx.Event()
 	rtm.SendMessage(rtm.NewOutgoingMessage(message, event.Channel))
```

Comment:
Could you please check if SendMessage returns an error? If so, return that instead of nill? Thank you!

Reply:
It does not return anything because it's sent through a channel https://github.com/slack-go/slack/blob/master/websocket.go#L88-L95

Reply:
Ok. Thanks for checking!